File: 0448 find-all-numbers-disappeared-in-an-array/solution.cs
public class Solution {
    public IList<int> FindDisappearedNumbers(int[] nums) {
        IList<int> ans=new List<int>();
        bool[] c=new bool[nums.Length+1];
        foreach(int i in nums) c[i]=true;
        for(int i=1;i<=nums.Length;i++) {
            if(!c[i]) ans.Add(i);
        }
        return ans;
    }
}

File: 0448 find-all-numbers-disappeared-in-an-array/solution.rs
impl Solution {
    pub fn find_disappeared_numbers(nums: Vec<i32>) -> Vec<i32> {
        let n = nums.len();
        let mut present = vec![false; n];

        for &i in &nums {
            present[(i - 1) as usize] = true;
        }

        let mut ans = Vec::new();
        for i in 1..=n {
            if !present[i - 1] {
                ans.push(i as i32);
            }
        }
        ans
    }
}
